Source/Core/DolphinQt/Config/Graphics/AdvancedWidget.cpp

```diff
@@ -185,13 +185,23 @@ void AdvancedWidget::CreateWidgets()
   dump_layout->addWidget(m_frame_dumps_resolution_type, 0, 1);
 
 #if defined(HAVE_FFMPEG)
-  m_dump_use_ffv1 =
-      new ConfigBool(tr("Use Lossless Codec (FFV1)"), Config::GFX_USE_FFV1, m_game_layer);
+  // Update EnableBitrate() if options are changed.
+
+  const std::vector<std::pair<QString, QString>> options{
+      {tr("Auto"), QStringLiteral("")},
```

Comment:
What does the `Auto` option do? It's not explained in the popup, and I personally don't think it's needed at all.

Reply:
It sets nothing, which is currently how it works. I'm not sure of how every OS handles that. It may be safe to not have Auto.

Reply:
If it's set to nothing then what happens when one tries to dump frames on `Auto`?